tiatoolbox/annotation/storage.py

```diff
@@ -1307,7 +1312,8 @@ def _query(
                 during the query.
 
         Returns:
-            sqlite3.Cursor: A database cursor for the current query.
```

Comment:
line 1308, `comparing`

Reply:
Sorry, but I don't understand this comment

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/3e63ea36471dbe1a6088ca317f39107c86a3c914/tiatoolbox/annotation/storage.py#L1319

Fix typo "comparing"